.github/workflows/copy-workflow.yml

```diff
@@ -150,6 +157,8 @@ jobs:
           # add DO NOT EDIT header
           tmp=$(mktemp)
           cat $TEMPLATE_REPO_DIR/$TEMPLATE_DIR/header.yml $TEMPLATE_REPO_DIR/$TEMPLATE_DIR/$f > $tmp
+          # replace $default-branch with this repo's GitHub default branch
+          sed -i "s/\$default-branch/${{ env.DEFAULTBRANCH }}/g" $tmp
```

Comment:
where is `$default-branch` written, though? don't we need that in the template yamls?

Reply:
Yes, that would be in the templates. Currently we don't use it, so this is a no-op for now.
Just opening this PR so we don't lose the code.